formats/json/commonMain/src/kotlinx/serialization/json/JsonElement.kt

```diff
@@ -315,6 +321,18 @@ public val JsonPrimitive.contentOrNull: String? get() = if (this is JsonNull) nu
 private fun JsonElement.error(element: String): Nothing =
     throw IllegalArgumentException("Element ${this::class} is not a $element")
 
+private inline fun <T> mapExceptionsToNull(f : () -> T) : T? {
+    return try { f()}
```

Comment:
formatting: space before brace.

I recommend using IntelliJ IDEA auto-formatter, as it honors Kotlin's style guide